bundles/org.openhab.binding.senechome/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.senechome-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
```

Comment:
```suggestion
    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
```

Reply:
Is there a way to avoid this change? For openhab3 it must be changed back to project again as per [guide-binding-development-changes-for-openhab-3-from-2-5-x](https://community.openhab.org/t/guide-binding-development-changes-for-openhab-3-from-2-5-x/104134)

Reply:
It was introduced specifically for 2.5.x because core is fixed. I assume with openHAB 3 we'll have core and addons keep in sync so would not be needed to have a different version and so it will be back to what it was. For bindings merged here this will be automatically done. So'll hope we can get this binding merged before 3.0.

Reply:
Thank you for the clarification! The change to ohc is already pushed. Is anything else missing for the merge?